packages/flutter/lib/src/widgets/app.dart

```diff
@@ -1310,7 +1308,7 @@ class _WidgetsAppState extends State<WidgetsApp> with WidgetsBindingObserver {
     final Map<String, Locale> languageAndCountryLocales = HashMap<String, Locale>();
     final Map<String, Locale> languageAndScriptLocales = HashMap<String, Locale>();
     final Map<String, Locale> languageLocales = HashMap<String, Locale>();
-    final Map<String, Locale> countryLocales = HashMap<String, Locale>();
+    final Map<String?, Locale> countryLocales = HashMap<String?, Locale>();
```

Comment:
do we really use null as a key in this map?

Reply:
`Local.countryCode` (used as key of the map) is nullable